src/index.js

```diff
@@ -1,6 +1,32 @@
-/**
-* Put your JavaScript here
-*/
+let state = {
+  todos: [
+    { title: 'First todo', complete: false },
+    { title: 'Second todo', complete: true },
+    { title: 'Third todo', complete: true },
+  ],
+};
+
+document.onload = init();
```

Comment:
I think I found the answer. It will work, but it's not the right thing to use because it would not fire until all elements have been loaded, including styles, scripts, and images: 

> The DOMContentLoaded event is fired when the initial HTML document has been completely loaded and parsed, without waiting for stylesheets, images, and subframes to finish loading. A very different event load should be used only to detect a fully-loaded page. It is an incredibly popular mistake to use load where DOMContentLoaded would be much more appropriate, so be cautious.

[source = Mozilla](https://developer.mozilla.org/en-US/docs/Web/Events/DOMContentLoaded)

True?